packages/mux-audio/README.md

```diff
@@ -44,13 +44,13 @@ require('@mux/mux-audio');
 Alternatively, use the CDN hosted version of this package:
 
 ```html
-<script src="https://unpkg.com/@mux/mux-audio@0.4"></script>
+<script src="https://cdn.jsdelivr.net/npm/@mux/mux-audio@0/dist/mux-audio.js"></script>
```

Comment:
if we wanted to, we can add a `jsdelivr` package field that points at this file to keep from needing to add the full path

Reply:
oh that would be neat, I'll add it to the PR later

Reply:
See https://www.jsdelivr.com/features#publishing-packages for more details

Reply:
👍 added this